### PR TITLE
When auth fails we clear the token from local storage

### PR DESCRIPTION
**Pull request by alexrothenberg**
This prevents some looping if we have a bad token -> snowflake login -> back here

### CODE REVIEW

**Comment by misfo** [comment]
Nice! :octocat:

Would this have prevent that epic redirect loop we ran into yesterday?

File: addon/mixins/authenticated-route-mixin.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ export default Ember.Mixin.create({
 
       (e) => {
         if(e.errors[0].status === '401') {
+          localStorage.removeItem('localStorage');
```
Comment (mattdoller) [inline addon/mixins/authenticated-route-mixin.js]: should this remove `access_token` instead of `localStorage`?

**Comment by alexrothenberg** [comment]
yes that is exactly what led me to this.

I think the reason we are hitting it is because we made the root application route use this authenticated mixin so when oauthd goes back to us in the /auth route it hits this before it updates the token itself.  I think this line works but we may also need to figure out how to ensure the "auth" route does not inherit from application

File: addon/mixins/authenticated-route-mixin.js (hunk 1 above)
Comment (alexrothenberg) [inline addon/mixins/authenticated-route-mixin.js]: oops yes you are totally right

**Pull request closed without merging**